src/dialogs/generic/show-dialog-box.ts

```diff
@@ -0,0 +1,62 @@
+import { fireEvent } from "../../common/dom/fire_event";
+
+interface AlertDialogParams {
+  confirmText?: string;
+  text?: string;
+  title?: string;
+  confirm?: (out?: string) => void;
+}
+
+interface ConfirmationDialogParams extends AlertDialogParams {
+  dismissText?: string;
+  cancel?: () => void;
+}
+
+interface PromptDialogParams extends AlertDialogParams {
+  inputLabel?: string;
+  inputType?: string;
+  default_value?: string;
```

Comment:
```suggestion
  defaultValue?: string;
```
Sorry, I know it was me 😇